24 - Adding SFTP to the protocols (#54)
The-Mycelium-Network/webworkbook: content/topics/protocols/index.md

- [Precisely glossary: FTPS](https://www.precisely.com/glossary/ftps)
- [Wikipedia: FTPS](https://en.wikipedia.org/wiki/FTPS)

> ### SFTP **SSH File Transfer Protocol**

Reviewer: ```suggestion
### SFTP (SSH File Transfer Protocol)
```

> SFTP functions pretty much the same as FTPS protocol, the difference is that SFTP makes use of the SSH protocol for security and authentication. SFTP is quickly taking the place of FTP and FTPS. The reason for this is that SFTP is more secure, more reliable and easier to configure. This means that there are not many reasons to use any other protocol in its place. SFTP is protected against *[man-in-the-middle-attacks](https://www.ssh.com/academy/attack/man-in-the-middle) and even *[password sniffing](https://www.ssh.com/academy/attack/password-sniffing).

Reviewer: ```suggestion
SFTP functions pretty much the same as FTPS protocol, both being protocols used to transfer files safely through an encrypted connection. The difference is the encryption type and the authentication process used. While FTPS just adds a security layer around the legacy FTP protocol, SFTP is an extension of the SSH protocol, which allows remote machines to verify each other before connecting and transferring data. 

SFTP is quickly taking the place of FTP and FTPS. The reason for this is that SFTP is more secure, more reliable and easier to configure. This means that there are not many reasons to use any other protocol in its place. SFTP is protected against [man-in-the-middle-attacks](https://www.ssh.com/academy/attack/man-in-the-middle) and even [password sniffing](https://www.ssh.com/academy/attack/password-sniffing).
```

**References:**

> * [SFTP-SSH Academy](https://www.ssh.com/academy/ssh/protocol)

Reviewer: ```suggestion
* [SFTP-SSH Academy](https://www.ssh.com/academy/ssh/protocol)
* [GeeksforGeeks: Difference between FTPS and SFTP](https://www.geeksforgeeks.org/difference-between-ftps-and-sftp/)
* [SmartFile: The Difference Between SFTP & FTPS and Why It Matters](https://www.smartfile.com/blog/difference-between-sftp-ftps/)
```
